webxrlayers-1.bs

```diff
@@ -1894,7 +1948,19 @@ Other layers types MUST continue to be sorted as before.
 
 {{XRQuadLayer}} and {{XRCylinderLayer}} layers MUST be sorted by their dimensions (for instance {{XRQuadLayerInit/width}} or {{XRCylinderLayerInit/centralAngle}}), transform and space.
 
-{{XRProjectionLayer}} layers MUST be sorted according to the values in their {{XRWebGLSubImage/depthStencilTexture}}. This also implies that if "[=depth-sorted-layers=]" is enabled, the [=XR Compositor=] MUST make use of depth values and {{XRProjectionLayer/ignoreDepthValues}} MUST be set to `true`.
+{{XRProjectionLayer}} layers MUST be sorted according to the values in their {{XRWebGLSubImage/depthStencilTexture}}. This also implies that if "[=depth-sorted-layers=]" is enabled, the [=XR Compositor=] MUST make use of depth values and {{XRProjectionLayer/ignoreDepthValues}} and {{XRWebGLBinding/usesDepthValues}} MUST be set to `true`.
+
+Space warp {#spacewarp}
+==========
+<dfn>Space warp</dfn> is a technology to that helps the [=XR Compositor=] do better prediction.
+By submitting a <dfn>motion texture</dfn> in {{XRWebGLSubImage}}'s {{XRWebGLSubImage/motionVectorTexture}} along with a {{XRWebGLSubImage/depthStencilTexture}}
+the [=XR Compositor=] has more information which results in improved rendering and better scene prediction.
```

Comment:
Also, the FB video covering this feature emphasizes that the primary use of this feature is it's ability to let the app run at half framerate. I suspect that's technically a separate "feature" from space warp, but I also suspect that there's going to be a high association between the two in developers minds.

This change doesn't mention anything about framerate throttling. Should it? And if not here, where?

Reply:
> Some more detail about what the motion texture should contain is needed here. 

o yes :-)

> Specifically, it's worth point out that the motion vector describes motion **from the previous frame to the current frame**, not the current frame to a future predicted frame.

No, the speed in the motion vector is the real world speed. There is delta pose that describes the difference between the last frame but I didn't expose that yet. (That delta is needed for locomotion)

Reply:
> This change doesn't mention anything about framerate throttling. Should it? And if not here, where?

I intentionally left that out because I wanted to leave it up to the UA to allow this. Maybe some systems would just use it for better reprojection or to double the available framerate (ie if they could hardware accelerate it somehow)